.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
I think that the state on which commit the submodule is, was saved during the commit when I added this submodule and updating it, requires manual action and another commit. But I need to read up on git submodule usage to be sure.